src/lib.rs

```diff
@@ -1251,6 +1242,225 @@ macro_rules! vob {
     });
 }
 
+impl<T: Debug + PrimInt> std::ops::BitOrAssign<&Vob<T>> for Vob<T> {
+    /// For each bit in this Vob, `or` it with the corresponding bit in `other`.
+    /// Store the result in `self`.
+    /// The two Vobs must have the same number of bits.
+    ///
+    /// # Panics
+    ///
+    /// If the two Vobs are of different length.
+    ///
+    /// # Examples
+    /// ```
+    /// use vob::vob;
+    /// let mut v1 = vob![true, false, true, false];
+    /// let v2 = vob![true, true, false, false];
+    /// v1 |= &v2;
+    /// assert_eq!(v1, vob![true, true, true, false]);
+    /// ```
+    #[inline(always)]
+    fn bitor_assign(&mut self, other: &Self) {
+        let _ = self.or(other);
+    }
+}
+
+impl<T: Debug + PrimInt> std::ops::BitOr<&Vob<T>> for &Vob<T> {
+    type Output = Vob<T>;
+    /// For each bit in this Vob, `or` it with the corresponding bit in `other`.
+    /// Store the result in clone of `self`.
+    /// The two Vobs must have the same number of bits.
+    ///
+    /// # Panics
+    ///
+    /// If the two Vobs are of different length.
+    ///
+    /// # Examples
+    /// ```
+    /// use vob::vob;
+    /// let v1 = vob![true, false, true, false];
+    /// let v2 = vob![true, true, false, false];
+    /// let v3 = &v1 | &v2;
+    /// assert_eq!(v3, vob![true, true, true, false]);
+    /// ```
+    #[inline(always)]
+    fn bitor(self, other: &Vob<T>) -> Vob<T> {
+        let mut rv = self.clone();
```

Comment:
Since this consumes `self`, can't we do `let _ = self.or(other)` as in `bitor_assign`?

Reply:
Ah, this is for `&Vob<T>`. Ignore my comment.